### DISCUSSION TITLE
Cannot install ArgoUML (Win11)

### DISCUSSION THREAD

**Discussion by grationd** [argouml-tigris-org/argouml#65] [Q&A] [answered]
Hello,

I'm on Windows 11 and whenever I try to install the program I'll get a pop up about downloading a JRE, but then the connection fails when it tries to download it from sun.com. However, I have installed the latest JDK from Oracle and the installer doesn't recognize it.

Any solutions?

![image](https://user-images.githubusercontent.com/99073678/212492467-d13199f4-6c4f-4132-a7ab-fd6527c24773.png)

![image](https://user-images.githubusercontent.com/99073678/212492519-5fed3c9d-7445-44a7-8f86-232cd2946f22.png)

**Comment by rfootq** [comment]
Hi, where you get installer? And question to @linustolke There are installer sources, how can I view them? Thanks

**Reply by grationd** [reply]
I'm using the exe file from the [releases page](https://github.com/argouml-tigris-org/argouml/releases) in this repo. I've tried both 0.34 and 0.35.1 versions with no luck.

**Reply by linustolke** [reply]
The installers are very old, as everything in this project. They were also made in a technique that we have since abandoned in favor of a maven-based approach and I think that isn't really in place yet.

**Reply by gerugeru** [reply]
how should we install? i have same issue as well

**Reply by Antoinnee** [reply, marked as answer]
Solution : Install the .zip file and execute the argouml.jar, instead of download the .exe setup file.
